styles.css

```diff
@@ -1,3 +1,37 @@
-body {
+@import url("https://fonts.googleapis.com/css?family=Lobster");
+
+/* .square {
+  padding: 10px;
+  margin: 10px;
+  background-color: white;
+  display: inline-block;
+  height: 25px;
+  width: 25px;
+  vertical-align: top;
+} */
+
+.board {
+  font-family: Lobster, monospace;
+  font-size: 40px;
+}
+
+.square {
```

Comment:
You can give your square a fixed width and height so the card name and suit can be displayed without changing the UI.
Eg.
```
width: 30px;
height: 30px;
```